lang/python-libatsha204/patches/0001-python-libatsha204-adjust-version-number-to-release-version.patch

```diff
@@ -0,0 +1,26 @@
+From cbccf6ed6ebb0d65f00ddb75997dffd44a8ddcc6 Mon Sep 17 00:00:00 2001
+From: Magnus Kessler <Magnus.Kessler@gmx.net>
+Date: Wed, 2 Aug 2023 09:08:27 +0200
+Subject: [PATCH] python: adjust version number to release version
+
+This is needed so that the correct python wheel archive is generated
```

Comment:
As well here. It is not advised to increase patches in Turris OS packages repository, when it can be included in upstream repository instead of patching it downstream while refreshing patches due to offset, fuzzes, etc.